package-res/resources/web/pentaho/lang/EventSource.js

```diff
@@ -325,10 +326,11 @@ define([
         result = this._emit(event);
       } catch(e) {
         if (event) {
```

Comment:
event always exist. You may put an error.argRequired error instead, if you want.